src/pages/Dashboard/Modals/SupplyWithdraw/SupplyWithdrawForm.tsx

```diff
@@ -77,23 +88,41 @@ export const SupplyWithdrawContent: React.FC<
       const temp = calculateNewBalance(amountInUsd);
       updateBorrowLimit = BigNumber.maximum(temp, 0);
     }
-
-    return [updateBorrowLimit];
+    const updateBorrowLimitCents = updateBorrowLimit?.multipliedBy(100);
+    return [updateBorrowLimitCents];
```

Comment:
Small thing: since you're returning only one value, no need to wrap it with an array.